cmd/session_test.go

```diff
@@ -35,9 +35,9 @@ func (s *TestSuite) TestSession(c *C) {
 	c.Assert(err, IsNil)
 	c.Assert(isSessionDirExists(), Equals, true)
 
-	session := newSessionV8()
+	session := newSessionV8(getHash("cp", []string{"mybucket", "myminio/mybucket"}))
```

Comment:
minor nit
```suggestion
	session := newSessionV8(getHash("cp", []string{"object", "myminio/mybucket"}))
```

Reply:
This is not correct. I am simulating `mc cp -r -c mybucket myminio/mybucket` here.